Porting FxCop Rules to Roslyn.md

```diff
@@ -0,0 +1,66 @@
+
+* Rules that do not relate to the usage of specific APIs, but relate instead to more general coding guidelines, should be organized according to the intended purpose of those guidelines. For example, some rules might help API authors produce consistent public APIs, but those rules might not make sense for test assemblies. (We will package those analyzers in Microsoft.ApiDesignGuidelines.Analyzers.dll, which by the way is essentially the original purpose of FxCop.) As another example, there might be some rules that restrict the expressiveness of the language (by discouraging the use of certain language features) in order to gain a performance advantage. Such rules would only apply in a specific context where that tradeoff is acceptable, and hence it would be useful to place them in a separate NuGet package.
+
+## Analyzer packages
+
+The list of all the rules that ship in VS, along with certain other FxCop/Roslyn rules that we know of, is captured in the file [RulesInventory.csv](https://github.com/dotnet/roslyn-analyzers/blob/master/RulesInventory.csv) file (which, thanks to GitHub, is searchable). That file also contains our proposed factoring of the analyzers (in the "Proposed Analyzer" column, which perhaps might have been better named "Proposed Analyzer Package").
+
+### API analyzer packages
+
+There are rules about types in the following contract assemblies:
+
+* **System.Runtime.Analyzers** - This package already exists
+
+* **System.Runtime.InteropServices.Analyzers** - Analyzers about interop and marshalling. This package already exists.
+
+* **System.Security.Cryptography.Algorithms** - Analyzers with guidelines for crypto algorithms usage. This is a new package.
+ 
+* **System.Xml** - Contains analyzers for types dealing with XML across  the System.Xml.* contracts. This is a new package.
+
+* **Desktop.Analyzers** - Contains analyzers for APIs that are present in the desktop .NET Framework but not in the new .NET Core API set. Since the .NET framework isn't available in a piecemeal fashion, there's not much value in breaking this down further.
+
+* **Microsoft.CodeAnalysis.Analyzers** - Contains rules about using the Roslyn APIs correctly. This package already exists.
+
+### Theme-based analyzer packages
+
+* **Microsoft.ApiDesignGuidelines** - Contains guidelines for authoring libraries which contain public APIs. The advantage of factoring it out this way is that one could simply install this analyzer for projects that expose real public APIs, and not for executables and test projects, reducing noise significantly.
+ 
```

Comment:
We've also added a Microsoft.QualityGuidelines.Analyzers now - we should probably list that as well.